perception/image_projection_based_fusion/src/pointpainting_fusion/node.cpp

```diff
@@ -140,6 +140,7 @@ PointPaintingFusionNode::PointPaintingFusionNode(const rclcpp::NodeOptions & opt
       isClassTable_.erase(cls);
     }
   }
+  has_variance_ = this->declare_parameter("has_variance", false);
```

Comment:
It's better not to use default variable so that we can clearly know whether the yaml parameter is used.
With this suggestion, we need to update autoware_launch first.

```suggestion
  has_variance_ = this->declare_parameter<bool>("has_variance");
```